tools/src/test/resources/adl2-tests/validity/rm_checking/openEHR-EHR-OBSERVATION.VCORMT_rm_non_conforming_type_use_node.v1.0.0.adls

```diff
@@ -0,0 +1,71 @@
+archetype (adl_version=2.0.5; rm_release=1.0.4)
```

Comment:
Is this archetype used anywhere?

Reply:
Everything in the validity directory is automatically used by the test. If it contains regression = <"SOME_ERROR">, the test asserts that that error is present. If "PASS", it must validate. If you remove the code changes, you will see this archetype causing a failed test.

Reply:
'the test' being com.nedap.archie.archetypevalidator.BigArchetypeValidatorTest :)

Reply:
Ah, I didn't know that. Nice method.